### ISSUE TITLE
Unable to Spin Up Komga Docker on DietPi

### ISSUE THREAD

**Issue by tuta23** [gotson/komga#1434]
### Steps to reproduce

Had a working version of Komga for sometime.  Recently upgraded the docker to the latest, and can no longer successfully deploy.  It appears to be an issue surrounding "Could not load libarchive.  no archive in java.library.path...."

At this point, I have been unable to google my way to a solution.

First line of the log shows "Starting ApplicationKt v1.10.3 using Java 21.0.1 with PID 1" if this is relevant.

### Expected behavior

Komga to successfully deploy 

### Actual behavior

Instead, it gets caught in a loop, and never finishes/completes.    GUI never becomes responsive either.

### Logs

2024-02-20T16:08:11.477Z  INFO 1 --- [main] org.gotson.komga.ApplicationKt           : Starting ApplicationKt v1.10.3 using Java 21.0.1 with PID 1 (/app/BOOT-INF/classes started by root in /app)                 
2024-02-20T16:08:11.551Z  INFO 1 --- [main] org.gotson.komga.ApplicationKt           : No active profile set, falling back to 1 default profile: "default"                                                         
2024-02-20T16:08:29.068Z  WARN 1 --- [main] c.github.gotson.nightcompress.Archive    : Could not load libarchive. no archive in java.library.path: :/usr/lib/x86_64-linux-gnu:
																																					/usr/java/packages/lib:
																																					/usr/lib64:/l>
2024-02-20T16:08:29.128Z  WARN 1 --- [main] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'rar5Configuration' since its singleton instance has been created too early. >

### Komga version

1.10.3

### Operating system

DietPi

### Installation method

Docker

### Acknowledgements

- [X] I have searched the existing issues (open AND closed) and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/docs/faq).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by DieselTech** [comment]
Looks like your pulling the x86_64 image when your likely running on ARM since you stated DietPi. What is the image name you have in your docker command?

**Comment by tuta23** [comment]
I'm using a template out of a Portainer instance - the relevant bits of the template seem to be:

"description": "Komga is a free and open source comics/mangas/magazines server.",
"image": "gotson/komga:latest",
"logo": "https://raw.githubusercontent.com/pi-hosted/pi-hosted/master/images/komga.png",
"name": "komga",
"platform": "linux",
"ports": [
"8995:8080/tcp"
			],
"restart_policy": "unless-stopped",
"title": "Komga",

**Comment by tuta23** [comment]
I changed the above port mapping to "25600:25600/tcp" and that seemed to allow the container to spin up completely.

This likely closes this issue, but attaching the log just to be complete.

The rest of the log - but this one was successful:
Version: 1.10.3
2024-02-21T14:35:17.472Z  INFO 1 --- [           main] org.gotson.komga.ApplicationKt           : Starting ApplicationKt v1.10.3 using Java 21.0.1 with PID 1 (/app/BOOT-INF/classes started by root in /app)
2024-02-21T14:35:17.520Z  INFO 1 --- [           main] org.gotson.komga.ApplicationKt           : No active profile set, falling back to 1 default profile: "default"
2024-02-21T14:35:32.797Z  WARN 1 --- [           main] c.github.gotson.nightcompress.Archive    : Could not load libarchive. no archive in java.library.path: :/usr/lib/x86_64-linux-gnu:/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
2024-02-21T14:35:32.829Z  WARN 1 --- [           main] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'rar5Configuration' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static' and/or marking the containing configuration class as 'proxyBeanMethods=false'.
2024-02-21T14:35:38.955Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : SqliteUdfPool - Starting...
2024-02-21T14:35:40.342Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : SqliteUdfPool - Added connection org.sqlite.jdbc4.JDBC4Connection@2591d23a
2024-02-21T14:35:40.353Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : SqliteUdfPool - Start completed.
2024-02-21T14:35:41.053Z  INFO 1 --- [           main] o.f.c.internal.license.VersionPrinter    : Flyway Community Edition 9.22.3 by Redgate
2024-02-21T14:35:41.055Z  INFO 1 --- [           main] o.f.c.internal.license.VersionPrinter    : See release notes here: https://rd.gt/416ObMi
2024-02-21T14:35:41.056Z  INFO 1 --- [           main] o.f.c.internal.license.VersionPrinter    : 
2024-02-21T14:35:41.153Z  INFO 1 --- [           main] org.flywaydb.core.FlywayExecutor         : Database: jdbc:sqlite:/config/database.sqlite (SQLite 3.45)
2024-02-21T14:35:41.572Z  INFO 1 --- [           main] o.f.core.internal.command.DbValidate     : Successfully validated 78 migrations (execution time 00:00.338s)
2024-02-21T14:35:41.615Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Current version of schema "main": 20231214163213
2024-02-21T14:35:41.632Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Schema "main" is up to date. No migration necessary.
2024-02-21T14:35:47.616Z  INFO 1 --- [           main] o.j.i.D.logVersionSupport                : Version                  : Database version is supported by dialect SQLITE: 3.45.0
2024-02-21T14:35:49.009Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port 25600 (http)
2024-02-21T14:35:49.054Z  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-02-21T14:35:49.055Z  INFO 1 --- [           main] o.apache.catalina.core.StandardEngine    : Starting Servlet engine: [Apache Tomcat/10.1.18]
2024-02-21T14:35:49.341Z  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2024-02-21T14:35:49.342Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 31256 ms
2024-02-21T14:35:52.624Z  WARN 1 --- [           main] c.g.g.n.j.i.plugins.JxlImageReaderSpi    : Could not load libjxl, plugin will be disabled. no jxl in java.library.path: :/usr/lib/x86_64-linux-gnu:/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
2024-02-21T14:35:52.626Z  INFO 1 --- [           main] c.g.g.n.j.i.plugins.JxlImageReaderSpi    : Deregistering service provider
2024-02-21T14:35:52.664Z  WARN 1 --- [           main] c.g.g.n.h.i.plugins.HeifImageReaderSpi   : Could not load libheif, plugin will be disabled. no heif in java.library.path: :/usr/lib/x86_64-linux-gnu:/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
2024-02-21T14:35:52.665Z  INFO 1 --- [           main] c.g.g.n.h.i.plugins.HeifImageReaderSpi   : Deregistering service provider
2024-02-21T14:35:52.698Z  WARN 1 --- [           main] c.g.g.n.w.i.plugins.WebpImageReaderSpi   : Could not load libwebp, plugin will be disabled. no webpdemux in java.library.path: :/usr/lib/x86_64-linux-gnu:/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib
2024-02-21T14:35:52.700Z  INFO 1 --- [           main] c.g.g.n.w.i.plugins.WebpImageReaderSpi   : Deregistering service provider
2024-02-21T14:35:52.909Z  INFO 1 --- [           main] o.g.k.i.image.ImageConverter             : Supported read formats: [JPG, JPEG 2000, tiff, PCX, bigtiff, wbp, bmp, gif, WBMP, PNG, RAW, JPEG, PNM, BigTIFF, tif, TIFF, wbmp, jpeg, jpeg-lossless, jbig2, jpg, JPEG2000, pcx, WBP, BMP, GIF, png, raw, BIGTIFF, JPEG-LOSSLESS, webp, JBIG2, pnm, TIF, jpeg2000, WEBP, jpeg 2000]
2024-02-21T14:35:52.913Z  INFO 1 --- [           main] o.g.k.i.image.ImageConverter             : Supported read mediaTypes: [image/vnd.wap.wbmp, image/jpeg, image/x-portable-graymap, image/bmp, image/x-windows-pcx, image/gif, image/x-pc-paintbrush, image/x-raw, image/webp, image/x-pcx, image/x-portable-bitmap, image/png, image/x-jb2, image/pcx, image/x-windows-bmp, image/x-webp, image/jpeg2000, image/jp2, image/x-bmp, image/x-png, image/x-portable-pixmap, image/tiff, image/x-tiff, image/x-jbig2, image/x-portable-anymap]
2024-02-21T14:35:52.918Z  INFO 1 --- [           main] o.g.k.i.image.ImageConverter             : Supported write formats: [JPEG 2000, JPG, tiff, bmp, PCX, bigtiff, gif, WBMP, PNG, RAW, JPEG, PNM, BigTIFF, tif, TIFF, wbmp, jpeg, jpg, JPEG2000, BMP, pcx, GIF, png, raw, BIGTIFF, pnm, TIF, jpeg2000, jpeg 2000]
2024-02-21T14:35:52.921Z  INFO 1 --- [           main] o.g.k.i.image.ImageConverter             : Supported write mediaTypes: [image/vnd.wap.wbmp, image/png, image/jpeg, image/x-portable-graymap, image/bmp, image/pcx, image/x-windows-bmp, image/x-windows-pcx, image/gif, image/jpeg2000, image/x-pc-paintbrush, image/x-raw, image/x-bmp, image/jp2, image/x-pcx, image/x-png, image/x-portable-bitmap, image/x-portable-pixmap, image/tiff, image/x-tiff, image/x-portable-anymap]
2024-02-21T14:35:53.798Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : SqliteTaskPool - Starting...
2024-02-21T14:35:53.802Z  INFO 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : SqliteTaskPool - Added connection org.sqlite.jdbc4.JDBC4Connection@39921410
2024-02-21T14:35:53.804Z  INFO 1 --- [           main] com.zaxxer.hikari.HikariDataSource       : SqliteTaskPool - Start completed.
2024-02-21T14:35:53.999Z  INFO 1 --- [           main] org.flywaydb.core.FlywayExecutor         : Database: jdbc:sqlite:/config/tasks.sqlite (SQLite 3.45)
2024-02-21T14:35:54.029Z  INFO 1 --- [           main] o.f.core.internal.command.DbValidate     : Successfully validated 1 migration (execution time 00:00.021s)
2024-02-21T14:35:54.039Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Current version of schema "main": 20231013114850
2024-02-21T14:35:54.042Z  INFO 1 --- [           main] o.f.core.internal.command.DbMigrate      : Schema "main" is up to date. No migration necessary.
2024-02-21T14:35:55.382Z  INFO 1 --- [           main] o.a.l.s.MemorySegmentIndexInputProvider  : Using MemorySegmentIndexInput with Java 21; to disable start with -Dorg.apache.lucene.store.MMapDirectory.enableMemorySegments=false
2024-02-21T14:36:03.550Z  INFO 1 --- [           main] o.s.b.a.w.s.WelcomePageHandlerMapping    : Adding welcome page: class path resource [public/index.html]
2024-02-21T14:36:06.874Z  INFO 1 --- [           main] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 18 endpoint(s) beneath base path '/actuator'
2024-02-21T14:36:07.357Z  INFO 1 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Will secure Or [Mvc [pattern='/api/**'], Mvc [pattern='/opds/**'], Mvc [pattern='/sse/**'], Mvc [pattern='/oauth2/authorization/**'], Mvc [pattern='/login/oauth2/code/**'], EndpointRequestMatcher includes=[*], excludes=[], includeLinks=true] with [org.springframework.security.web.session.DisableEncodeUrlFilter@4b391e45, org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@647bd553, org.springframework.security.web.context.SecurityContextHolderFilter@7abeaed0, org.springframework.security.web.header.HeaderWriterFilter@5d956649, org.springframework.web.filter.CorsFilter@54602d5a, org.springframework.security.web.authentication.logout.LogoutFilter@f1b0658, org.springframework.security.web.session.ConcurrentSessionFilter@72c8de78, org.springframework.security.web.authentication.www.BasicAuthenticationFilter@7bb6a1f4, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@2ddb260e, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@482040a, org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter@25bd8eab, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@161974d1, org.springframework.security.web.session.SessionManagementFilter@7536ef0f, org.springframework.security.web.access.ExceptionTranslationFilter@347074a9, org.springframework.security.web.access.intercept.AuthorizationFilter@598f065f]
2024-02-21T14:36:10.765Z  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port 25600 (http) with context path ''
2024-02-21T14:36:10.789Z  INFO 1 --- [           main] org.gotson.komga.ApplicationKt           : Started ApplicationKt in 56.163 seconds (process running for 58.397)
2024-02-21T14:36:10.937Z  INFO 1 --- [         task-1] g.k.i.s.ThumbnailMetadataFixerController : Find and fix thumbnails without metadata
2024-02-21T14:36:10.962Z  INFO 1 --- [         task-1] o.g.komga.application.tasks.TaskEmitter  : Sending task: FixThumbnailsWithoutMetadata(priority='0')
2024-02-21T14:36:11.144Z  INFO 1 --- [   scheduling-1] .AuthenticationActivityCleanupController : Remove authentication activity older than 2024-01-21T14:36:11.128355583 (UTC)
2024-02-21T14:36:11.764Z  INFO 1 --- [         task-7] o.g.k.i.scheduler.SearchIndexController  : Lucene index version: 8
2024-02-21T14:36:11.784Z  INFO 1 --- [io-25600-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-02-21T14:36:11.787Z  INFO 1 --- [io-25600-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2024-02-21T14:36:11.798Z  INFO 1 --- [io-25600-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 9 ms
2024-02-21T14:36:12.714Z  INFO 1 --- [taskProcessor-1] o.g.komga.application.tasks.TaskHandler  : Executing task: FixThumbnailsWithoutMetadata(priority='0')
2024-02-21T14:36:12.862Z  INFO 1 --- [taskProcessor-1] o.g.k.domain.service.ThumbnailLifecycle  : Fetched 0 ThumbnailBook to fix, total: 0
2024-02-21T14:36:12.896Z  INFO 1 --- [taskProcessor-1] o.g.k.domain.service.ThumbnailLifecycle  : Fixed 0 ThumbnailBook in 151.932518ms
2024-02-21T14:36:12.934Z  INFO 1 --- [taskProcessor-1] o.g.k.domain.service.ThumbnailLifecycle  : Fetched 0 ThumbnailSeries to fix, total: 0
2024-02-21T14:36:12.938Z  INFO 1 --- [taskProcessor-1] o.g.k.domain.service.ThumbnailLifecycle  : Fixed 0 ThumbnailSeries in 31.512963ms
2024-02-21T14:36:12.974Z  INFO 1 --- [taskProcessor-1] o.g.k.domain.service.ThumbnailLifecycle  : Fetched 0 ThumbnailSeriesCollection to fix, total: 0
2024-02-21T14:36:12.979Z  INFO 1 --- [taskProcessor-1] o.g.k.domain.service.ThumbnailLifecycle  : Fixed 0 ThumbnailSeriesCollection in 27.387129ms
2024-02-21T14:36:13.010Z  INFO 1 --- [taskProcessor-1] o.g.k.domain.service.ThumbnailLifecycle  : Fetched 0 ThumbnailReadList to fix, total: 0
2024-02-21T14:36:13.015Z  INFO 1 --- [taskProcessor-1] o.g.k.domain.service.ThumbnailLifecycle  : Fixed 0 ThumbnailReadList in 25.135259ms
2024-02-21T14:36:13.020Z  INFO 1 --- [taskProcessor-1] o.g.komga.application.tasks.TaskHandler  : Task FixThumbnailsWithoutMetadata(priority='0') executed in 298.281426ms
